xrpl/transaction/__init__.py

```diff
@@ -1,11 +1,11 @@
 """Top-level exports for the transaction package."""
+from xrpl.models.transactions.transaction import transaction_json_to_binary_codec_form
```

Comment:
I know this is bad form but I want to remove `transaction_json_to_binary_codec_form` from the public API at some point - I don't think it's necessary now that `to_xrpl` does that work. If this is the wrong way to handle this, I'm happy to change that

Reply:
instead of this, i think it would be clearer if you kept `transaction_json_to_binary_codec_form` but just had that call your new function